addons/scratchstats/addon.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "Profile statistics",
-  "description": "Shows statistics from ScratchDB on user profiles.",
+  "description": "Shows statistics on user profiles.",
   "tags": ["community", "profiles"],
   "credits": [
     {
       "name": "Maximouse",
       "link": "https://scratch.mit.edu/users/Maximouse/"
-    }
+    },
+    { "name": "ScratchDB", "url": "http://scratchdb.lefty.one/v3/docs/", "note": "for information", "id": "scratchdb" }
```

Comment:
Why one-liner?

Reply:
I was lazy and I thought prettier would fix it lol